### PR TITLE
Change bundle ID to app.jamulussoftware.*

### LINKED ISSUE #1029: Update macOS path with new bundle id
io.jamulus.jamulus must be updated 
to app.jamulussoftware.Jamulus throughout the whole page due to the bundle ID change. 
_https://github.com/jamulussoftware/jamuluswebsite/pull/1019#discussion_r1730117998_

### PR DESCRIPTION
**Pull request by ann0see**
**Short description of changes**
Changes the server and client bundle id to `app.jamulussoftware.*`

**Context: Fixes an issue? Related issues**
Fixes: #1029

**Status of this Pull Request**
Ready for review

**What is missing until this pull request can be merged?**
Review and potential translation fixes

**Does this need translation?**

YES, but all translations have been replaced.

## Checklist

- [x] I've verified that this Pull Request follows the [general code principles](https://github.com/jamulussoftware/jamulus/blob/main/CONTRIBUTING.md#jamulus-projectsource-code-general-principles)
- [x] I waited some time after this Pull Request was opened and all GitHub checks completed without errors. 
- [x] I'm sure that this Pull Request goes to the correct branch

### CODE REVIEW

File: _translator-files/po/nl/Include-Shared-Commands.po [hunk 1]
```diff
@@ -27,8 +27,8 @@ msgstr "`-h` of `--help` Laat de help tekst zien"
...
+msgid "`-i` or `--inifile` Set location of initialization file (overrides default. On macOS simply provide a filename only, since config files can only be read from `/Users/<username>/Library/Containers/app.jamulussoftware.Jamulus/Data/`. For the server replace `app.jamulussoftware.Jamulus` with `app.jamulussoftware.JamulusServer`. Turn on \"Show Library folder\" in \"Show view options\" in Finder to see this folder.)"
+msgstr "`-i` of `--inifile` Stel de locatie van het initialisatiebestand in (heeft voorrang op de standaardinstelling. Geef op macOS alleen een bestandsnaam op, aangezien configuratiebestanden alleen kunnen worden gelezen vanuit `/Users/<gebruikersnaam>/Library/Containers/io.jamulus .Jamulus/Data/`. Vervang voor de server `app.jamulussoftware.Jamulus` door `app.jamulussoftware.JamulusServer`. Schakel \"Toon bibliotheekmap\" in \"Toon weergaveopties\" in Finder in om deze map te zien.)"
```
Comment (ann0see) [inline _translator-files/po/nl/Include-Shared-Commands.po]: There's something missing!
```suggestion
msgstr "`-i` of `--inifile` Stel de locatie van het initialisatiebestand in (heeft voorrang op de standaardinstelling. Geef op macOS alleen een bestandsnaam op, aangezien configuratiebestanden alleen kunnen worden gelezen vanuit `/Users/<gebruikersnaam>/Library/Containers/app.jamulussoftware.Jamulus/Data/`. Vervang voor de server `app.jamulussoftware.Jamulus` door `app.jamulussoftware.JamulusServer`. Schakel \"Toon bibliotheekmap\" in \"Toon weergaveopties\" in Finder in om deze map te zien.)"
```

**Review by ann0see — COMMENTED** [review]
Self review looks ok, but still please check for oddities.

File: wiki/en/Software-Manual.md [hunk 2]
```diff
@@ -293,7 +293,7 @@ Applies a one-off fader setting to each channel depending on its volume. Useful
 
 **Note for macOS users:** As of Jamulus 3.8.1, we have a signed installer. This will store the settings in 
 ```shell
-$HOME/Library/Containers/io.jamulus.Jamulus/Data/.config/Jamulus/
+$HOME/Library/Containers/app.jamulussoftware.Jamulus/Data/.config/Jamulus/
```
Comment (pljones) [inline wiki/en/Software-Manual.md:296]: Should this be marked "no translate" in some way?

**Review by ignotus666 — APPROVED** [review]
Looks ok to me.

File: _translator-files/po/de/Software-Manual.po [hunk 3]
```diff
@@ -804,19 +804,19 @@ msgstr "Du kannst verschiedene Mixereinstellungen mit [Laden/Speichern Konfigura
...
 #| "```shell\n"
-#| "$HOME/Library/Containers/io.jamulus.Jamulus/Data/.config/Jamulus/\n"
+#| "$HOME/Library/Containers/app.jamulussoftware.Jamulus/Data/.config/Jamulus/\n"
 #| "```\n"
 msgid "**Note for macOS users:** As of Jamulus 3.8.1, we have a signed installer. This will store the settings in \n"
 msgstr ""
 "**Hinweis für macOS-Nutzer:** Ab Jamulus 3.8.1 gibt es ein signiertes Installationsprogramm. Dieser speichert die Einstellungen in\n"
 "```shell\n"
-"$HOME/Library/Containers/io.jamulus.Jamulus/Data/.config/Jamulus/\n"
+"$HOME/Library/Containers/app.jamulussoftware.Jamulus/Data/.config/Jamulus/\n"
```
Comment (pljones) [inline _translator-files/po/de/Software-Manual.po:813]: These three copies appear to be duplicates (lines 801 through 820 need cleaning up).

**Comment by pljones** [comment]
Nothing blocking.

**Review by pljones — APPROVED** [review]

File: _translator-files/po/de/Software-Manual.po (hunk 3 above)
Comment (ann0see) [reply _translator-files/po/de/Software-Manual.po:813]: I think it’s translation memory. We could clean it but there’s no need.

File: wiki/en/Software-Manual.md (hunk 2 above)
Comment (ann0see) [reply wiki/en/Software-Manual.md:296]: Yes.

Comment (ann0see) [reply wiki/en/Software-Manual.md:296]: Done on weblate.

**Pull request merged**